RNTester/RNTester-macOS/TurboModuleCxx/winrt/Microsoft.ReactNative.h

```diff
@@ -5,6 +5,12 @@
 namespace winrt::Microsoft::ReactNative
 {
 
+template <typename T>
+decltype(auto) ReadOptional(std::optional<T> &opt)
+{
+  return opt.__get();
+}
```

Comment:
Does this need a `#ifndef __APPLE__` guard as well?

Reply:
No, this file only exist in react-native-macos.